templates/nginx/nginx.conf.template

```diff
@@ -143,6 +143,7 @@ http {
         proxy_set_header  Host            $host;
         proxy_set_header  X-Real-IP       $remote_addr;
         proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header  Connection      $http_connection;
```

Comment:
Can we forward the upgrade header too? this was a header that was also missing when i was testing locally (and aleph/netty will complain if it's missing)